### PR TITLE
adding tests for empty signer simulation

### PR DESCRIPTION
**Pull request by barnjamin**

### CODE REVIEW

File: algosdk/atomic_transaction_composer.py [hunk 1]
```diff
@@ -676,6 +676,19 @@ def sign_transactions(
         pass
 
 
+class EmptySinger(TransactionSigner):
```
Comment (algochoi) [inline algosdk/atomic_transaction_composer.py:679]:
```suggestion
class EmptySigner(TransactionSigner):
```

Comment (barnjamin) [reply algosdk/atomic_transaction_composer.py]: ah nice, just fixed locally thanks!
